src/features/HallLayout/hooks/useDeleteHallRow.ts

```diff
@@ -0,0 +1,16 @@
+import { useParams } from "react-router-dom";
+import { deleteHallRow } from "../service/hallLayoutService";
+
+export const useDeleteHallRow = () => {
+    const params = useParams();
+
+    const deleteHallRowHandler = async (rowId: string) => {
+        const hallId = params.hallId;
+        if(hallId)
+        await deleteHallRow(hallId, rowId);
+    };
+
+    return {
+      deleteHallRowHandler
+    }
```

Comment:
Add ; after returned object and add a trailing comma since you already sterted using it in other parts of the code